src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const KEY in robot) {
+    const VALUE = robot[KEY];
```

Comment:
Good Job😎. However, I am not sure whether it is a good practice to name all variables in that style. Please, could you explain why did you name them like that?

Reply:
Thank you. Only now I saw an inconsistency in naming my 'const inversedRobot' )))
But this is a good question you've asked! Capitalizing constants, I was following Misha's affirmation here (at 2m 14s):
https://mate.academy/learn/javascript-advanced/solving-tasks-on-github?section=video&videoId=5119&testTaskSlug=js_transportation_on_vacation
However, I haven't seen him go down this path anywhere other than here, and our linter doesn't require anything. So, I think we'll wait and see how it goes later)...

Reply:
Other than that, I think I found good but aomewhat controversial explanations here:

https://learn.javascript.ru/task/uppercast-constant

https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/const
![image](https://github.com/mate-academy/js_inverse_robot/assets/110335429/f3f6234c-51ff-4ed9-92e8-3c3bc2ae0d39)


https://ru.hexlet.io/topics/32587#:~:text=%D0%97%D0%B4%D0%B5%D1%81%D1%8C%20%D0%B2%D0%BE%D0%BF%D1%80%D0%BE%D1%81%20%D0%B7%D0%B0%D0%BA%D0%BB%D1%8E%D1%87%D0%B0%D0%B5%D1%82%D1%81%D1%8F%20%D0%B2%20%D1%81%D1%82%D0%B8%D0%BB%D0%B5,%D0%B8%D0%BC%D0%B5%D0%BD%D0%B0%20%D0%BE%D0%B1%D1%8B%D1%87%D0%BD%D1%8B%D1%85%20%D0%BA%D0%BE%D0%BD%D1%81%D1%82%D0%B0%D0%BD%D1%82%20%D0%B1%D0%BE%D0%BB%D1%8C%D1%88%D0%B8%D0%BC%D0%B8%20%D0%B1%D1%83%D0%BA%D0%B2%D0%B0%D0%BC%D0%B8.
![image](https://github.com/mate-academy/js_inverse_robot/assets/110335429/44ecfb1b-14a3-44e9-b936-1fcd64cb075b)

Reply:
And there's another decent comment, this time from our mentor:

![image](https://github.com/mate-academy/js_inverse_robot/assets/110335429/bc370fed-24fe-4cd9-b7d5-f10071d93555)

Reply:
I returned all constant names back to normal